bundles/pubsub/pubsub_api/include/pubsub/subscriber.h

```diff
@@ -39,7 +39,7 @@
 #define PUBSUB_SUBSCRIBER_SCOPE                "scope"
 #define PUBSUB_SUBSCRIBER_CONFIG               "pubsub.config"
 
-#define PUBSUB_SUBSCRIBER_SCOPE_DEFAULT        "default"
+//#define PUBSUB_SUBSCRIBER_SCOPE_DEFAULT        "default"
```

Comment:
Why comment out the subscriber but not the publisher?

Reply:
Leftovers, fixed.